manifests/init.pp

```diff
@@ -161,6 +163,7 @@
   Variant[Enum['unmanaged'], Stdlib::Absolutepath] $defaultzonepath = $dns::params::defaultzonepath,
   Optional[Enum['only', 'first']] $forward                          = undef,
   Array[Dns::Forwarder] $forwarders                                 = [],
+  Array $rpz_zones                                                  = [],
```

Comment:
what's the data in the array? Always an Fqdn? Or we could use Stdlib::Fqdn?
```suggestion
  Array[Stdlib::Fqdn] $rpz_zones                                    = [],
```

https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/types/fqdn.pp

Reply:
think I've got a valid basic test in place now